activerecord/test/cases/associations_test.rb

```diff
@@ -41,6 +41,28 @@ def test_subselect
     assert_equal favs, fav2
   end
 
+  def test_clear_association_cache_stored
+    firm = Firm.find(1)
+    assert_kind_of Firm, firm
+
+    firm.send(:clear_association_cache)
```

Comment:
Whoops, that's from reverting the commit that removed this (which was after the method was privatized). Thanks.